pages/data/dataset_info/_id.vue

```diff
@@ -179,29 +194,35 @@ export default {
       } else {
         this.dataset = this.$route.params.id
       }
+
       const discoveryMetadataURL =
         this.$config.WOUDC_UI_API_URL + '/collections/discovery_metadata/items'
-      this.uriDatasetDef = discoveryMetadataURL + '?identifier=' + this.dataset
+
+      this.uriDatasetDef = discoveryMetadataURL + '/' + this.dataset
```

Comment:
```suggestion
      this.uriDatasetDef = `${discoveryMetadataURL}/${this.dataset}&f=json`
```

Mainly to ensure `f=json` that we're requesting